### ISSUE TITLE
Error - Process finished with exit code -1073740791 (0xC0000409)

### ISSUE THREAD

**Issue by TomerEphraim** [OlafenwaMoses/ImageAI#620]
Hello,

I'm trying to train a detection model on some 512x384 color images, using a pretrained yolov3 model.
This is the code I run:

```
from imageai.Detection.Custom import DetectionModelTrainer

trainer = DetectionModelTrainer()
trainer.setModelTypeAsYOLOv3()
trainer.setDataDirectory(data_directory=images_folder)
trainer.setTrainConfig(object_names_array=object_names, batch_size=4, num_experiments=3, train_from_pretrained_model="pretrained-yolov3.h5")
trainer.trainModel()
```

The train and validation subfolders inside _images_folder_ contain about 5 images each.
When I run the code on pycharm, several seconds after the usual tensorflow outputs, I get this message:

`Process finished with exit code -1073740791 (0xC0000409)`

I thinks it has something to do with the gpu memory running out of space.
I checked the system monitor and the gpu memory is indeed full.

If I disable the GPU then it works just fine.
Any idea how can run this on the GPU without getting this error?

I am using the latest imageai version (2.16) with all the required modules installed (Including all the CUDA related stuff):
Python 3.7
tensorflow 2.4.0
Keras 2.3.0
numpy 1.19.3
keras-resnet 0.2.0
scipy 1.4.1 
h5py 2.10.0 
matplotlib 3.3.2 
opencv-python

Additional Details:
IDE: Pycharm
OS: Windows 10 Pro
GPU: Nvidia GTX 1080 ti
CPU: Intel Core-i7 8th gen

**Comment by OlafenwaMoses** [comment]
@TomerEphraim 
- have you tried setting the **batch_size** to 2 and running the training? This value highly influences the GPU usage of your training.
- Also, try running the training from **Powershell** . E.g if your python file is `detection_training.py`, navigate to the folder containing this file via Powershell and run the command below

  ```
  python detection_training.py
  ```

**Comment by TomerEphraim** [comment]
@OlafenwaMoses thanks for the reply, I've eventually solved this issue.

I've tried to run the training from the command line as you suggested, and I saw some error messages that were not displayed when I ran it from Pycharm. I guess this is because these messages are not python related, and thus not shown in Pycharm output.

The additional error messages where related to additional missing cuda includes, libs and dlls, like "cudnn_ops_train.*".
I guess these files are not directly used by tensorflow, otherwise tensorflow would notify about this in its output.

Anyway, thanks for the help!
